app/src/forms/admin/routes.js

```diff
@@ -65,18 +367,141 @@ routes.get('/users/:userId', async (req, res, next) => {
 //Form componets help info
 //
 
+/**
+ * @openapi
+ * /admin/formcomponents/proactivehelp/object:
```

Comment:
Oh, it's great that you found endpoints that weren't documented before. This will be very helpful in the future!